src/preferences/dialog/dlgprefwaveform.cpp

```diff
@@ -49,6 +49,11 @@ DlgPrefWaveform::DlgPrefWaveform(
         defaultZoomComboBox->addItem(QString::number(100 / static_cast<double>(i), 'f', 1) + " %");
     }
 
+    m_pOverviewMinuteMarkersControl = std::make_unique<ControlObject>(
+            ConfigKey(QStringLiteral("[Waveform]"),
+                    QStringLiteral("OverviewMinuteMarkers")));
```

Comment:
I'm sorry for the naming nitpicks, but I think we should add a verb here, also making it consistent with the variable identifiers is nicer. 
```suggestion
    m_pOverviewMinuteMarkersControl = std::make_unique<ControlObject>(
            ConfigKey(QStringLiteral("[Waveform]"),
                    QStringLiteral("DrawOverviewMinuteMarkers")));
```